app/views/welcome/index.html.erb

```diff
@@ -1,77 +1,213 @@
+      <div class="card-image">
+        <%= image_tag "Newsletter.jpg"%>
+        <span class="card-title"><h5>Newsletters</h5></span>
+      </div>
+      <div class="card-content white-text">
+        <p>Send newsletters and notifications to your guests reminding them about the event</p>
+      </div>
+      <div class="card-action">
+        <a href="#">Learn More</a>
       </div>
     </div>
   </div>
-
-  <%= render "welcome/homepage_tab", text_id: "featured", example_id: "featured_content" %>
-  <%= render "welcome/homepage_tab", text_id: "popular", example_id: "popular_content" %>
 </div>
 
-<div class="near-footer">
-  <h4>Create your own event</h4>
-
-  <p>Bring people together, or turn your passion into a business.
-    <br>
-    EventX gives you everything you need to host your best event yet.
-  </p>
-
-    <% if can? :create, Event %>
-      <%= create_event_or_login("Get Started", ["waves-effect", "btn-large bottom_btn z-depth-0"]) %>
-    <% else %>
-      <%= become_a_manager("Get Started", ["waves-effect", "btn-large bottom_btn z-depth-0"]) %>
-    <% end %>
-
+<div class="row homepage-cards">
+  <div class="col m4">
+    <div class="card teal darken-1 small">
+      <div class="card-image">
+        <%= image_tag "promote-event.jpg"%>
+        <span class="card-title"><h5>Promote your Events</h5></span>
+      </div>
+      <div class="card-content white-text">
+        <p>Use our powerful social media tools to increase attendance and enhance attendee engagement</p>
+      </div>
+      <div class="card-action">
+        <a href="#">Learn More</a>
+      </div>
+    </div>
+  </div>
+  <div class="col m4">
+    <div class="card teal darken-1 small">
+      <div class="card-image">
+        <%= image_tag "manage-events.jpg"%>
+        <span class="card-title"><h5>Manage your Events</h5></span>
+      </div>
+      <div class="card-content white-text">
+        <p>Our tools allow you to be in the driver seat and take charge of your events</p>
+      </div>
+      <div class="card-action">
+        <a href="#">Learn More</a>
+      </div>
+    </div>
+  </div>
+  <div class="col m4">
+    <div class="card teal darken-1 card small">
+      <div class="card-image">
+        <%= image_tag "recommend.jpg"%>
+        <span class="card-title"><h5>Feedback on Events</h5></span>
+      </div>
+      <div class="card-content white-text">
+        <p>Get ratings and reviews from guests about how they felt about your events</p>
+      </div>
+      <div class="card-action">
+        <a href="#">Learn More</a>
+      </div>
+    </div>
+  </div>
 </div>
+
+<div id="search-events" class="yellow darken-1">
+  <br><br><h4 class="center">DO WHAT YOU LOVE</h4><br>
+  <h5 class="center">Find an event near you</h5>
+    <div class="col l12">
+    <div class="row our-search">
+      <div class="header center search_form our-search l12">
+        <%= render "welcome/search" %>
+      </div>
+    </div>
+  </div>
```

Comment:
Nitpick - I think we can refactor this file into partials, especially the new cards added.